api/FileSystemHandle.json

```
@@ -189,6 +189,41 @@
           }
         }
       },
+      "remove": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FileSystemHandle/remove",
+          "support": {
+            "chrome": {
+              "version_added": "110"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```

```suggestion
            "experimental": false,
            "standard_track": true,
```
I'm not too familiar with the format here, but it seems like these bits should be flipped? This is being standardized in [whatwg/fs](https://github.com/whatwg/fs/pull/9)

---

"Experimental" should be left as `true` because only one browser engine, Chromium, supports it.  Also, we don't set "Standard Track" to `true` until it has landed in a spec's standards track, so the pull request must be merged first.

---

Cheers for providing the explanation @queengooborg 

@a-sully I'm happy to update this as soon as it lands in the spec.